src/sqlfluff/rules/L017.py

```diff
@@ -9,17 +9,19 @@
 class Rule_L017(BaseRule):
     """Function name not immediately followed by bracket.
```

Comment:
Suggest changing the wording, as "bracket" is internal parser terminology. We should also update the wording on line 33.
```suggestion
    """Function name not immediately followed by parenthesis.
```

Reply:
Think it's more common in UK English. But happy to accept for consistency.